### PR TITLE
Fix pdfrenderer.py reference

### PR DESCRIPTION
**Pull request by tfmorris**
Thanks for preserving the license btw!

### CODE REVIEW

**Comment by MerlijnWajer** [comment]
Thanks, merged

File: README.rst [hunk 1]
```diff
@@ -215,7 +215,7 @@ License
 
 License for all code (minus ``internetarchive/pdfrenderer.py``) is AGPL 3.0.
 
-``internetarchive/pdfrenderer.py`` is Apache 2.0, which matches the Tesseract
+``internetarchivepdf/pdfrenderer.py`` is Apache 2.0, which matches the Tesseract
```
Comment (mara004) [inline README.rst:218]: wait, the line above too

File: README.rst [hunk 2]
```diff
@@ -215,7 +215,7 @@ License
 
 License for all code (minus ``internetarchive/pdfrenderer.py``) is AGPL 3.0.
```
Comment (mara004) [inline README.rst:216]: here

**Comment by MerlijnWajer** [comment]
Just saw that too, I can make an additional commit.

**Comment by mara004** [comment]
Yeah. Low prio though, after all it's clear what is meant.

**Comment by MerlijnWajer** [comment]
Done

**Comment by tfmorris** [comment]
Thanks for the quick merge! Sorry I missed the other reference.

**Pull request merged**